netpoll_test.go

```diff
@@ -64,6 +65,13 @@ func Assert(t *testing.T, cond bool, val ...interface{}) {
 	}
 }
 
+var testPort int32 = 10000
+
+// getTestAddress return a unique port for every tests, so all tests will not share a same listerner
+func getTestAddress() string {
+	return fmt.Sprintf("127.0.0.1:%d", atomic.AddInt32(&testPort, 1))
```

Comment:
actually we can listen on 127.0.0.1:0 and let os picks an available port.

Reply:
@xiaost if so , I need to use listener.Address.String to get the real port and for current tests, it need change too many test codes, because listener is hidden by newTestEventLoop now.